ti-community-bots/config/database_config.yaml.example

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
```

Comment:
It should be example.yaml not yaml.example.

Reply:
This is to prevent the example file from being applied.

Reply:
It's not readable. Maybe we can move it into examples.